### PR TITLE
Auto update for Mac

### PR DESCRIPTION
**Pull request by MortalKastor**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "productName": "Ledger Live",
   "description": "Ledger Live - Desktop",
   "repository": "https://github.com/LedgerHQ/ledger-live-desktop",
-  "version": "0.1.0",
+  "version": "0.1.0-alpha",
```
Comment (meriadec) [inline package.json]: I think we should use `1.0.0-alpha.x`

File: src/main/autoUpdate.js [hunk 2]
```diff
@@ -16,5 +17,14 @@ export default (notify: SendFunction) => {
...
 
 export function quitAndInstall() {
-  autoUpdater.quitAndInstall()
+  setImmediate(() => {
+    const browserWindows = BrowserWindow.getAllWindows()
+
+    app.removeAllListeners('window-all-closed')
+    browserWindows.forEach(browserWindow => {
+      browserWindow.removeAllListeners('close')
+    })
```
Comment (meriadec) [inline src/main/autoUpdate.js]: curious about what was the problem here?

Comment (gre) [reply src/main/autoUpdate.js]: (add a comment in sourcecode)

**Comment by MortalKastor** [comment]
Successfully built and drafted a release (in this repo) which should self-update on each new `prerelease`.

**Comment by meriadec** [comment]
Naiiiiiiceu

**Pull request merged**